### PR TITLE
[ru_RU] Add jobTitle

### PR DESCRIPTION
**Pull request by 2bj**

### CODE REVIEW

File: src/Faker/Provider/ru_RU/Company.php [hunk 1]
```diff
@@ -25,6 +25,61 @@ class Company extends \Faker\Provider\Company
         'Текстиль'
     );
 
+    protected static $jobTitleFormat = array(
```
Comment (fzaninotto) [inline src/Faker/Provider/ru_RU/Company.php]: what's the datasource? Is it MIT-compatible?

Comment (2bj) [inline src/Faker/Provider/ru_RU/Company.php]: List compiled myself from different sources

Comment (fzaninotto) [inline src/Faker/Provider/ru_RU/Company.php]: Then you must list the datasources and they must all be MIT compatible

**Comment by fzaninotto** [comment]
No news, closing.

**Pull request closed without merging**